server.go

```diff
@@ -31,12 +33,26 @@ type quoteResponse struct {
 	cyrptokey string
 }
 
+var (
+	delayVal = rand.Intn(4) + 1
```

Comment:
Yes, but individual response times are no longer random because `delayVal` never changes after it's called the first time.